susemanager/src/mgr_bootstrap_data.py

```diff
@@ -538,6 +538,11 @@
     "dmidecode",
 ]
 
+PKGLISTUBUNTU2004 = [
+    "salt-common",
+    "salt-minion",
```

Comment:
There will be more packages, don't panic.

I want to start from scratch instead of reusing stuff from Ubuntu 18.04, so we can have only the packages we really need for a really minimal Ubuntu installation (I am trying with Ubuntu Server without anything preinstalled but the openssh server).

Reply:
Ok, so I tried this and... looks just too good. I was able to bootstrap a minimal Ubuntu 20.04 minion only with this. No need to copy anything at all from Universe.

Too good to be true. I need to review this tomorrow.